test/Microsoft.ML.CodeGenerator.Tests/Microsoft.ML.CodeGenerator.Tests.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
```

Comment:
You don't need this. See how we switch the TFM in:

[https://github.com/dotnet/machinelearning/blob/master/test/Directory.Build.props#L5-L7](https://github.com/dotnet/machinelearning/blob/master/test/Directory.Build.props#L5-L7)